crates/precompile/Cargo.toml

```diff
@@ -15,7 +15,7 @@ reth-revm.workspace = true
 alloy-primitives.workspace = true
 
 p256 = { version = "0.13.2", features = ["ecdsa"] }
-
+revm-precompile = { git = "https://github.com/bluealloy/revm.git", features = ["secp256r1"] }
```

Comment:
this has been available since at least revm-precompiles 11, so we don't need this

Reply:
I felt using crates.io version was against the project dependency rules , so I used the git , I can change it to crates version